app/UserPreferences.m

```diff
@@ -133,20 +146,30 @@ - (instancetype)init {
     self = [super init];
     if (self) {
         _defaults = [NSUserDefaults standardUserDefaults];
-        Theme *defaultTheme = [Theme presetThemeNamed:@"Light"];
         [_defaults registerDefaults:@{
-            kPreferenceFontFamilyKey: @"Menlo",
             kPreferenceFontSizeKey: @(12),
-            kPreferenceThemeKey: defaultTheme.properties,
             kPreferenceCapsLockMappingKey: @(CapsLockMapControl),
             kPreferenceOptionMappingKey: @(OptionMapNone),
             kPreferenceBacktickEscapeKey: @(NO),
             kPreferenceDisableDimmingKey: @(NO),
             kPreferenceLaunchCommandKey: @[@"/bin/login", @"-f", @"root"],
             kPreferenceBootCommandKey: @[@"/sbin/init"],
+            kPreferenceBlinkCursorKey: @(NO),
+            kPreferenceCursorStyleKey: @(CursorStyleBlock),
             kPreferenceHideStatusBarKey: @(NO),
+            kPreferenceColorSchemeKey: @(ColorSchemeMatchSystem),
+            kPreferenceThemeKey: @"Default",
         }];
-        _theme = [[Theme alloc] initWithProperties:[_defaults objectForKey:kPreferenceThemeKey]];
+        // https://webkit.org/blog/10247/new-webkit-features-in-safari-13-1/
+        if (@available(iOS 13.4, *)) {
+            [_defaults registerDefaults:@{
```

Comment:
This appears to be redundant with -approximateFont, why not always set this to kSystemMonospacedFontName and let approximateFont change that to menlo as needed?

Reply:
`kSystemMonospacedFontName` is morally only supposed to support ui-monospaced, which is the SF Mono font. That -approximateFont returns Menlo if you happen to set this value before 13.4 is really just there as a backstop.